src/agent/libclusterfuzz/third-party/update.sh

```diff
@@ -12,6 +12,7 @@ git clone --depth 1 https://github.com/google/clusterfuzz clusterfuzz-src
 mv clusterfuzz-src/src/python/lib/clusterfuzz/stacktraces/constants.py .
 mkdir -p ../data/stack-traces
 cp clusterfuzz-src/src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_data/*.txt ../data/stack-traces/
+chmod -x ../data/stack-traces/*.txt
```

Comment:
What's going on here?

Reply:
some portion of the text files in the clusterfuzz repo are executable.  this fixes that.

Reply:
Oh wow, I misread the `-` as a `+`. That makes _way_ more sense.